### DISCUSSION TITLE
Do the top languages stats support private repositories?

### DISCUSSION THREAD

**Discussion by andrewhong04** [anuraghazra/github-readme-stats#1716] [Q&A] [answered]
Do the top languages stats support private repositories?

**Comment by rickstaa** [comment, marked as answer]
Hey @novialriptide. Thanks for your question. It is indeed possible for the top language card to include data from private repositories. To do this, Vercel has to have permission to fetch your data. As a result, the public Vercel, which does not have permission to access users' private data, only shows data from public repositories. 

By deploying a Vercel instance yourself and adding a personal access token (see https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance), you can get private data to show up in the stats card.
